app/controllers/ui_ansible_roles_controller.rb

```diff
@@ -7,6 +9,16 @@ def index
     @ui_ansible_roles = resource_scope_for_index(:permission => :view_ansible_roles)
   end
 
+  api :DELETE, '/ui_ansible_roles/:id', N_('Deletes Ansible role')
+  param :id, :identifier, :required => true
+  def destroy
+    role = AnsibleRole.find_by!(id: params[:id])
+    if role.nil?
```

Comment:
`find_by!` raises an exception, so this code is unreachable

Reply:
Ah, I was wondering about that. Apparently, the RecordNotFound-Exception still produces a 404 error, so this isn't needed anyways.